src/dev/impl/DevToys/Strings/fr-FR/Common.resw

```diff
@@ -168,6 +168,9 @@
   <data name="Megabytes" xml:space="preserve">
     <value>MO</value>
   </data>
+  <data name="Now" xml:space="preserve">
+    <value>À présent</value>
```

Comment:
I'd suggest to keep the word in English for language that you may not be fluent with, and let people who are comfortable with the language translating it. Here is an example: while `À présent` is correct literal translation of `Now`, in this context, it's incorrect and is confusing. The correct one would be `Maintenant`.

Reply:
Fixed in [cbf2329](https://github.com/veler/DevToys/pull/515/commits/cbf2329c2a35103e35fc16bb30ba6228c47445ad).

I wasn't sure if English or Google Translate was preferred in contributions. Now I know. 👍